_episodes/13-ides.md

```diff
@@ -66,116 +66,103 @@ and some can also interact with a version control system.
 Compared to an IDE, a good dedicated code editor is usually smaller and quicker,
 but often less feature-rich.
 You will have to decide which one is the best for you -
-in this course we will learn how to use [PyCharm](https://www.jetbrains.com/pycharm/),
+in this course we will learn how to use [VS Code](https://code.visualstudio.com/),
 a free, open source Python IDE.
 Some popular alternatives include
 free and open source IDE [Spyder](https://www.spyder-ide.org/)
-and Microsoft's free [Visual Studio Code (VS Code)](https://code.visualstudio.com/). 
+and JetBrain's free [PyCharm](https://www.jetbrains.com/pycharm/). 
 
 > ## Using VS Code for This Course 
-> If you want to use VS Code as your IDE for this course, there is a separate [extras episode](../vscode/index.html)
-> to help you set up. The instructions for PyCharm in the course will not apply to you verbatim but there 
-> is an equivalent functionality in VS Code for each of the actions we ask you to do in PyCharm.
+> If you want to use PyCharm as your IDE for this course, there is a separate [extras episode](../pycharm/NEEDS UPDATING.html)
```

Comment:
Perhaps we should point the readers to the original course here, while that is still using PyCharm, to save us the work of creating an extras episode for this?

Reply:
Agreed. Do you have the correct link to the original course? I probably have it on my bookmarks, but I want to make sure I get the correct one.

Reply:
I've changed this to give the link to the IDE lesson in the original course:
https://carpentries-incubator.github.io/python-intermediate-development/13-ides/index.html